synapse/models/entity.py

```diff
@@ -0,0 +1,16 @@
+import synapse.lib.module as s_module
+
+class EntityModule(s_module.CoreModule):
+
+    def getModelDefs(self):
+        return (('entity', {
+
+            'types': (
+                ('entity:name', ('str', {'onespace': True, 'lower': True, 'strip': True}), {
```

Comment:
@invisig0th I added the ``strip`` opt here too. I can easily back it out if you want.

Reply:
onespace is a super-set of strip, so it's redundant. Remove plz :)